src/toxcore/dht/dht_node.rs

```diff
@@ -114,22 +90,14 @@ impl DhtNode {
             saddr_v6,
             last_resp_time_v4,
             last_resp_time_v6,
-        }
-    }
-
-    /// calc. status of node
-    pub fn calc_status(&self, bad_node_timeout: Duration) -> NodeStatus {
-        if self.last_resp_time_v4.elapsed() > bad_node_timeout &&
-            self.last_resp_time_v6.elapsed() > bad_node_timeout {
-            NodeStatus::Bad
-        } else {
-            NodeStatus::Good
+            last_ping_req_time: None,
         }
     }
 
     /// check it the node is timed out
-    pub fn is_bad_node_timed_out(&self, server: &Server) -> bool {
-        self.calc_status(Duration::from_secs(server.config.bad_node_timeout)) == NodeStatus::Bad
```

Comment:
Add comment: the node is bad only when it does not answer both on ipv4 and ipv6 addrs for N secods